.github/workflows/tests-ui.yml

```diff
@@ -16,30 +16,10 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Build Dockerfile
-        id: build-client-image
-        uses: docker/build-push-action@v4
-        with:
-          builder: ${{ steps.buildx.outputs.name }}
-          cache-from: type=gha,scope=cal-itp
-          cache-to: type=gha,scope=cal-itp,mode=max
```

Comment:
FYI, I don't think the caching was working, and even if it was (in some cases?), this change actually speeds up the build:

<img width="1025" alt="Screenshot 2023-02-15 at 10 50 37 PM" src="https://user-images.githubusercontent.com/86842/219264172-276e12a0-5dc1-4cc9-aa29-6875d090f059.png">

Reply:
If the caching isn't working, then the main deploy workflow should be looked at too, it has this exact same sequence of steps.